x-pack/test/observability_functional/apps/observability/exploratory_view.ts

```diff
@@ -48,16 +42,28 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await esArchiver.unload(
         Path.join('x-pack/test/apm_api_integration/common/fixtures/es_archiver', 'rum_8.0.0')
       );
+      await esArchiver.unload(
```

Comment:
Just curious: if in `loadIfNeeded` the archive is not loaded, how does `unload` work in that case?